expected-consensus.md

```diff
@@ -204,7 +204,7 @@ In the case that everybody draws a losing ticket in a given round (i.e. no miner
 
 In addition to each attempted `ElectionProof` generation, the miner will need to extend the ticket chain by generating a new ticket. They use the ticket they generated in the prior round, rather than the prior block's (as is normally used). This proves appropriate delay (given that finding a winning Ticket has taken multiple rounds). Thus, each time it is discovered that nobody has won in a given round, every miner should use their previously generated ticket to repeat the ticket generation process, appending said ticket to their would-be block's `Ticket` array. This continues until some miner finds a winning ticket (see below), ensuring that the ticket chain remains at least as long as the block chain.
 
-TODO: FIX As was stated above, in the case where the tickets at round `N-K` are all losing tickets, i.e. if no block was generated in round `N-K`, miners should attempt to generate their `ElectionProof` from that ticket which eventually leads to the creation of the block in the next TipSet with the smallest winning ticket, rather than the smallest block at round `N-K` (both are equivalent but this makes implementation simpler).
+TODO: @why to review As was stated above, in the case where there are multiple tickets to choose from at round `N-K` (i.e. if the TipSet eventually created has multiple blocks), miners should attempt to generate their `ElectionProof` from the ticket generated by the block with the smallest final ticket (i.e. not necessarily the smallest ticket generated at that round).
```

Comment:
this sounds technically correct, let's add a TODO to have a diagram to illustrate the problem